### ISSUE TITLE
[Feature request] add possibility to display tasks synchronized from davx5

### ISSUE THREAD

**Issue by Djey51** [Etar-Group/Etar-Calendar#1354]
Hi,

Could you add the possibility to display the tasks synchronized through davx5 please?

Thanks

**Comment by jspricke** [comment]
See #1279

**Comment by jspricke** [comment]
Task support is not yet implemented, see #1279 for the current state.

**Comment by SamuXzX** [comment]
@jspricke yeah, I understand. I'm just confused by what you anwered: Opentask is not Davx5. Dadx5 lets you manage tasks integrating with different applications: one of them is OpenTasks, another one is Tasks.org.

@Djey51 asked about Davx5, not OpenTasks. I'm trying to understand why, given that Davx5 doesn't identify with OpenTasks, would you point to the thread about OpenTasks.

Or mayne I'm confusing things and they actually are the same thing.

**Comment by jspricke** [comment]
Similar to the explanation in the Readme: https://github.com/Etar-Group/Etar-Calendar/tree/88b3b79f901b3368596bbeeefe9c45970b8e06e9#technical-explanation

Etar is an app to view events (and with #1279 to view as well tasks). For that to work there needs to be a provider for the data (for example OpenTasks) and optionally one to sync them (for example Davx5).

**Comment by SamuXzX** [comment]
I understand. Would Tasks.org work as a provider? Is it needed to work to integrate a new provider with Etar or the integration for one provider works for every provider?

**Comment by jspricke** [comment]
To my understanding it will add support for both but did not find time to look into it yet.

**Comment by SamuXzX** [comment]
Thank you very much

**Comment by MrRulf** [comment]
but why is an extra app needed, arent the tasks stored in calendars? meaning etar could access them trough the same way it accesses other calendar data? i know there are task only, calendar only and files that can store both, but since etar accesses those files anyway for calendar, why not add task support aswell? (this is a question and suggestion cuz i dont know the code, im not saying do that and its urgent, i know its an open source project etc. I'm even considering doing it on my own, or at least trying, as i doubt having the necessary skills, but before i invest lots of time only to figure out that my idea is not possible, or not trough this way, or does not follow a principle this project follows, bc showing tasks directly would not be kiss because it might be beyond the scope of a calendar, i rather ask that and save a lot of time ^^")

**Comment by jspricke** [comment]
They are not stored in the same place. Also this is not the right place to discuss architecture questions.
